package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "quickdraw-dom",
-  "version": "0.10.18",
+  "version": "0.10.19",
```

Comment:
:thinking_face: Since we’re adding a new property, does this qualify for a minor version bump? Or does the fact that we’re still at 0.x override that with the implicit suggestion that the interface may change suddenly?

Reply:
Yeah, making this a minor version bump makes sense to me! Honestly, I just let the gulp task auto-increment the version 😅